tools/cli/cmd/root.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
```

Comment:
Please fill in the copyright

Reply:
Removed template copyright comment - already have LICENSE in the root of the repo